src/Parsers/ASTCreateQuery.h

```diff
@@ -14,6 +15,30 @@ class ASTFunction;
 class ASTSetQuery;
 class ASTSelectWithUnionQuery;
 
+/// DEFINER = <user_name | CURRENT_USER> SQL SECURITY <DEFINER | INVOKER | NONE>
+/// If type was not set during parsing, the default type from settings will be used.
+/// Currently supports only views.
+class ASTSQLSecurity : public IAST
```

Comment:
I know we have ASTStorage here but still maybe it's better to move ASTSQLSecurity to another file.